File: Heat/Migrations/201508291936347_BoilerHeater_CertificationReference.vb
Imports System
Imports System.Data.Entity.Migrations
Imports Microsoft.VisualBasic

Namespace Migrations
    Public Partial Class BoilerHeater_CertificationReference
        Inherits DbMigration
    
        Public Overrides Sub Up()
            AddColumn("dbo.BoilerHeaters", "CertificationReference", Function(c) c.String())
            AddColumn("dbo.BoilerServices", "PreviousServiceDate", Function(c) c.DateTime(nullable := False))
            AddColumn("dbo.BoilerServices", "Periodicity", Function(c) c.Int(nullable := False))
            AddColumn("dbo.BoilerServices", "LegalExpirationDate", Function(c) c.DateTime(nullable := False))
            AddColumn("dbo.BoilerServices", "PlannedServiceDate", Function(c) c.DateTime(nullable := False))
        End Sub
        
        Public Overrides Sub Down()
            DropColumn("dbo.BoilerServices", "PlannedServiceDate")
            DropColumn("dbo.BoilerServices", "LegalExpirationDate")
            DropColumn("dbo.BoilerServices", "Periodicity")
            DropColumn("dbo.BoilerServices", "PreviousServiceDate")
            DropColumn("dbo.BoilerHeaters", "CertificationReference")
        End Sub
    End Class
End Namespace

File: Heat.ConvertedToC#/Migrations/201508291936347_BoilerHeater_CertificationReference.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Linq;
using System.Xml.Linq;
using System.Diagnostics;
using System.Collections.Specialized;
using System.Configuration;
using System.Text;
using System.Text.RegularExpressions;
using System.Web;
using System.Web.Caching;
using System.Web.Mvc;
using System.Web.Mvc.Ajax;
using System.Web.Mvc.Html;
using System.Web.Routing;
using System.Web.SessionState;
using System.Web.Security;
using System.Web.Profile;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Web.UI.HtmlControls;
using System.Data.Entity.Migrations;

namespace Heat.Migrations
{
	public partial class BoilerHeater_CertificationReference : DbMigration
	{

		public override void Up()
		{
			AddColumn("dbo.BoilerHeaters", "CertificationReference", c => c.String());
			AddColumn("dbo.BoilerServices", "PreviousServiceDate", c => c.DateTime(nullable: false));
			AddColumn("dbo.BoilerServices", "Periodicity", c => c.Int(nullable: false));
			AddColumn("dbo.BoilerServices", "LegalExpirationDate", c => c.DateTime(nullable: false));
			AddColumn("dbo.BoilerServices", "PlannedServiceDate", c => c.DateTime(nullable: false));
		}

		public override void Down()
		{
			DropColumn("dbo.BoilerServices", "PlannedServiceDate");
			DropColumn("dbo.BoilerServices", "LegalExpirationDate");
			DropColumn("dbo.BoilerServices", "Periodicity");
			DropColumn("dbo.BoilerServices", "PreviousServiceDate");
			DropColumn("dbo.BoilerHeaters", "CertificationReference");
		}
	}
}
